AWSAuthSDK/Tests/AWSMobileClientTests/AWSMobileClientUserAttributeTests.swift

```diff
@@ -130,7 +130,7 @@ class AWSMobileClientUserAttributeTests: AWSMobileClientTestBase {
         }
 
         wait(for: [getAttrExpectation], timeout: 5)
-
+        AWSMobileClient.default().removeUserStateListener(self)
```

Comment:
should we do this in other tests as well? might make them all more stable

Reply:
Not all test uses the `addUserStateListener` and some test might need it throughout the test lifecycle. We need to audit and figure this out, and is out of scope for this test fix.